uiplib/scheduler.py

```diff
@@ -5,7 +5,13 @@
 import time
 from uiplib.scrape import get_images
 from threading import Thread
-
+import sys
+from select import select
+try:
+    import msvcrt
+except ImportError:
+    #not on windows
```

Comment:
No msvcrt module is available only on windows :(
But I have defined functions for mac and linux